cmd/root.go

```diff
@@ -55,8 +53,32 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&env.Port, "ssh-port", "p", "", "OCP3 ssh port")
 
 	// Get config file from CLI argument an save to viper config
-	rootCmd.PersistentFlags().StringP("output-dir", "o", path.Dir("."), "set the directory to store extracted configuration.")
+	rootCmd.PersistentFlags().StringP("output-dir", "o", "", "set the directory to store extracted configuration.")
 	env.Config().BindPFlag("OutputDir", rootCmd.PersistentFlags().Lookup("output-dir"))
+
+	// Set log level from CLI argument
+	rootCmd.PersistentFlags().String("config-source", "", "source for OCP3 config files, accepted values: remote or local")
+	env.Config().BindPFlag("ConfigSource", rootCmd.PersistentFlags().Lookup("config-source"))
+
+	// Get crio config file location
+	rootCmd.PersistentFlags().String("crio-config", "", "path to crio config file")
```

Comment:
Can we make the default OCP 3.x values the default value for CPMA?

Same for the other ones: master-config, node-config, etc.

Reply:
Yes, default values are set only in case of usage ssh as source for config

Reply:
https://github.com/fusor/cpma/pull/279/files#diff-1e94282afa1637126c37c2d26509a2d4R259